src/auto/AutoModule.lua

```diff
@@ -55,11 +55,13 @@ return function(moduleName)
       end
 
       function module:accGradParameters(input, gradOutput, scale)
-         if not self.grads then
-            self.grads = self.b({input=input, weight=self.weight, bias=self.bias}, gradOutput)
+         if self.weight then
```

Comment:
I think you need to be more careful with that, there is at least one module in nn which has bias but no weight, see https://github.com/torch/nn/blob/master/Add.lua#L9